credential-controller.js

```diff
@@ -159,6 +159,9 @@ function factory(
         if(!('identity' in session) && self.credential.sysIsPublic) {
           // display a public credential to a non-authenticated user
           _display('credentialInfo');
+        } else if('CREDENTIAL_ADMIN' in session.identity.sysPermissionTable) {
```

Comment:
Actually, that check is already happening as well:
https://github.com/digitalbazaar/bedrock-angular-credential/blob/adminPermission/credential-controller.js#L159

Merging this now.

On Thu, Apr 21, 2016 at 1:12 PM, Dave Longley <notifications@github.com>
wrote:

> In credential-controller.js
> <https://github.com/digitalbazaar/bedrock-angular-credential/pull/4#discussion_r60619622>
> :
>
> > @@ -159,6 +159,9 @@ function factory(
> >          if(!('identity' in session) && self.credential.sysIsPublic) {
> >            // display a public credential to a non-authenticated user
> >            _display('credentialInfo');
> > +        } else if('CREDENTIAL_ADMIN' in session.identity.sysPermissionTable) {
>
> Would you like to add a check for that prior to this entire subroutine?
>
> If it's possible to get back a session without an identity, yes.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/digitalbazaar/bedrock-angular-credential/pull/4/files/c324a301f91e1440b07e21fc2b61208c27e63584#r60619622>
>



-- 
*Matt Collier*